win/sneedacity.rc

```diff
@@ -31,21 +31,21 @@ BEGIN
         BEGIN
 #ifdef EXPERIMENTAL_DA        
             VALUE "CompanyName", "Extended Sneedacity Team"
-            VALUE "FileDescription", "A customised Sneedacity� Cross-Platform Sound Editor"
+            VALUE "FileDescription", "A customised Sneedacity Cross-Platform Sound Editor"
             VALUE "FileVersion", SNEEDACITY_FILE_VERSION
             VALUE "InternalName", "DarkSneedacity"
             VALUE "LegalCopyright", "Copyright � 2018. All rights reserved."
-            VALUE "LegalTrademarks", "Sneedacity� is a registered trademark of Dominic Mazzoni."
+            VALUE "LegalTrademarks", "Sneedacity is not a registered trademark (at least yet)."
```

Comment:
Are we planning to trademark the sneed? ;)

Reply:
you can just delete this line

Reply:
Yeah, I'd say comment out everything that is not relevant as long as it doesn't break the build